frontend/app/src/pages/assets/[identifier].vue

```diff
@@ -2,6 +2,7 @@
 import { type RawLocation } from 'vue-router';
 import { type ComputedRef } from 'vue';
 import { type AssetBalanceWithPrice } from '@rotki/common';
+import { useSupportedChains } from '@/composables/info/chains';
```

Comment:
remove the import

Reply:
😄 damn we all have the same problem with Webstorm, I wonder how much effort it would be to add a linter check for this